src/wp-admin/options-general.php

```diff
@@ -107,34 +107,75 @@
 
 	$classes_for_upload_button = 'upload-button button-add-media button-add-site-icon';
 	$classes_for_update_button = 'button';
+	$classes_for_wrapper       = '';
 
-	$classes_for_avatar = 'avatar avatar-150';
 	if ( has_site_icon() ) {
-		$classes_for_avatar          .= ' has-site-icon';
+		$classes_for_wrapper         .= ' has-site-icon';
 		$classes_for_button           = $classes_for_update_button;
 		$classes_for_button_on_change = $classes_for_upload_button;
 	} else {
-		$classes_for_avatar          .= ' hidden';
+		$classes_for_wrapper         .= ' hidden';
 		$classes_for_button           = $classes_for_upload_button;
 		$classes_for_button_on_change = $classes_for_update_button;
 	}
 
+	// Handle alt text for site icon on page load.
+	$site_icon_id           = (int) get_option( 'site_icon' );
+	$app_icon_alt_value     = '';
+	$browser_icon_alt_value = '';
+
+	if ( $site_icon_id ) {
+		$img_alt            = get_post_meta( $site_icon_id, '_wp_attachment_image_alt', true );
+		$filename           = wp_basename( get_site_icon_url() );
+		$app_icon_alt_value = sprintf(
+			/* Translators: %s: The selected image filename. */
```

Comment:
Nit: we always write the translator comments with lowercase t.

```suggestion
			/* translators: %s: The selected image filename. */
```

Reply:
Resolved in https://github.com/WordPress/wordpress-develop/pull/6165/commits/e8f7676765fbafd74305dfcd2a3cf2bac7c49295